test/config/DocumentationControllerSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import api.controllers.ControllerBaseSpec
+import com.typesafe.config.ConfigFactory
+import config.rewriters._
+import controllers.{AssetsConfiguration, DefaultAssetsMetadata, RewriteableAssets}
+import definition.ApiDefinitionFactory
+import mocks.MockAppConfig
+import org.scalatest.OptionValues
+import play.api.Configuration
+import play.api.http.{DefaultFileMimeTypes, DefaultHttpErrorHandler, FileMimeTypesConfiguration, HttpConfiguration}
+import play.api.mvc.Result
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class DocumentationControllerSpec extends ControllerBaseSpec with MockAppConfig with OptionValues {
```

Comment:
Is it also worth unit testing the `definition` method in the controller?

Reply:
It is tested via the ITs so I don't think so (and it is a pretty simple method).